### ISSUE TITLE
TabView: Unable to select closable icon of tabs through keyboard

### ISSUE THREAD

**Issue by akshayaqburst** [primefaces/primereact#7437]
### Describe the bug

TabView: Unable to select closable icon of tabs through keyboard

### System Information

```Shell
System:
    OS: Linux 5.15 Ubuntu 20.04.6 LTS (Focal Fossa)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
    Memory: 4.57 GB / 7.47 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 18.20.2 - ~/.nvm/versions/node/v18.20.2/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v18.20.2/bin/yarn
    npm: 10.5.0 - ~/.nvm/versions/node/v18.20.2/bin/npm
    pnpm: 9.6.0 - ~/.nvm/versions/node/v18.20.2/bin/pnpm
  npmPackages:
    react: 18.3.1 => 18.3.1
```

### Steps to reproduce the behavior

1. Go to https://primereact.org/tabview/#closable
2. Use right arrow key to navigate between Tabs.
3. Close Icon is not getting focussed on right arrow key or tab press.

### Expected behavior

The close icon should be focussable through keyboard navigation and user should be able to dismiss it.
